api.md

```diff
@@ -1669,14 +1669,15 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
          [ optional grouping exprs>]
 [HAVING ...]
 ```
+Note that a single hypertable is supported in the `FROM` clause.
 
 #### Parameters
 |Name|Description|
 |---|---|
 | `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
 | `<column_name>`| Optional list of names to be used for columns of the view. If not given, the column names are deduced from the query.|
 | `WITH` clause | This clause specifies [options](#continuous_aggregate-create_view-with) for the continuous aggregate view.|
-| `<select_query>`| A `SELECT` query that uses the specified syntax. |
+| `<select_query>`| A `SELECT` query that uses the specified syntax, which allows only a singe hypertable in the `FROM` clause. |
```

Comment:
Let's not make this change.  Two reminders is enough, and very long parameter args just are hard to read.  Plus, the "suggestion" as written is actually confusing.

Reply:
Removed